addon/styles/frost-theme.scss

```diff
@@ -20,13 +21,36 @@ $frost-color-input-disabled-bg: $frost-color-lgrey-3 !global;
 $frost-color-selection: $frost-color-blue-1 !global;
 $frost-color-button-focus-shadow: rgba($frost-color-blue-1, .75) !global;
 
-// Select
+// == Select ==================================================
+
 $frost-color-select-container-background: $frost-color-white !global;
 $frost-color-select-container-shadow: $frost-color-grey-5 !global;
 $frost-color-select-footer-divider: $frost-color-blue-1 !global;
 $frost-color-select-indicator: $frost-color-grey-5 !global;
 $frost-color-select-item-divider: $frost-color-lgrey-1 !global;
 
+// == Checkbox ================================================
+
+$frost-checkbox-border-default: $frost-color-grey-6 !default;
```

Comment:
I know it's annoying, but to avoid overlaps in the variable namespace I'd prefix all of these with `color` - e.g. $frost-color-....